.github/workflows/publish-release-bte.yml

```diff
@@ -0,0 +1,113 @@
+name: Publish Release Candidates BTE
+
+on:
+  push:
+    branches:
+      - bte-release
```

Comment:
So the idea here is that instead of pushing a build every time someone merges something into develop, we actively choose when to release a build to the stores by fast forward merging the bte-release and gps-release branches.  This will save us from having a lot extra builds submitted for processing that we are actually intending to ship.

Reply:
superb